tools/integration_tests/util/setup/setup.go

```diff
@@ -146,11 +146,11 @@ func CompareFileContents(t *testing.T, fileName string, fileContent string) {
 	}
 }
 
-func CreateTempFile() string {
+func CreateTempFile(dirPath string) string {
```

Comment:
Nit: There is a similar method in file_operations.go, shall we use it instead? I also think that we should pass filepath to this method as passing dirName is not very intuitive.